deepcell/data/__init__.py

```diff
@@ -30,33 +30,49 @@
 from __future__ import print_function
 
 
-def split_dataset(dataset, validation_data_fraction):
+def split_dataset(dataset, val_data_fraction, test_data_fraction=0):
     """
-    Splits a dataset of type tf.data.Dataset into a training and validation
-    dataset using given ratio. Fractions are rounded up to two decimal places.
+    Splits a dataset of type tf.data.Dataset into a training, validation, and
+    optionally test dataset using given ratios. Fractions are rounded up to
+    two decimal places.
 
     Inspired by: https://stackoverflow.com/a/59696126
 
     Args:
         dataset (tf.data.Dataset): the input dataset to split.
-        validation_data_fraction (float): the fraction of the validation data
+        val_data_fraction (float): the fraction of the validation data
             between 0 and 1.
+        test_data_fraction (float): the fraction of the test data between 0
+            and 1.
 
     Returns:
-        (tf.data.Dataset, tf.data.Dataset): a tuple of (training, validation).
+        (tf.data.Dataset, tf.data.Dataset, tf.data.Dataset): a tuple of
+            (training, validation, test).
     """
-    validation_data_percent = round(validation_data_fraction * 100)
-    if not (0 <= validation_data_percent <= 100):
-        raise ValueError('validation_data_fraction must be ∈ [0,1].')
+    val_data_percent = round(val_data_fraction * 100)
+    if not 0 <= val_data_percent <= 100:
+        raise ValueError('val_data_fraction must be ∈ [0,1].')
+
+    test_data_percent = round(test_data_fraction * 100)
+    if not 0 <= test_data_percent <= 100:
+        raise ValueError('test_data_fraction must be ∈ [0,1].')
+
+    if val_data_percent + test_data_percent >= 100:
+        raise ValueError('sum of val_data_fraction and '
+                         + 'test_data_fraction must be ∈ [0,1].')
```

Comment:
There is no need to concatenate these strings, as they are inside the parentheses. This will be inferred as a multi-line string:

```suggestion
        raise ValueError('sum of val_data_fraction and '
                         'test_data_fraction must be ∈ [0,1].')
```